tests/integration/pipeline/base.py

```diff
@@ -0,0 +1,119 @@
+import os
+import re
+import shutil
+from pathlib import Path
+from typing import List
+from unittest import TestCase
+
+import boto3
+
+from tests.testing_utils import run_command_with_input
+
+
+class PipelineBase(TestCase):
+    def base_command(self):
+        command = "sam"
+        if os.getenv("SAM_CLI_DEV"):
+            command = "samdev"
+
+        return command
```

Comment:
[out of scope of this PR] This method along with other many methods are duplicated in all of the integrations test. We need to have a Base class for all of them.